os/network-config-with-networkd.md

```diff
@@ -1,8 +1,8 @@
 # Network Configuration with networkd
 
-CoreOS machines are preconfigured with [networking customized]({{site.baseurl}}/docs/sdk-distributors/distributors/notes-for-distributors) for each platform. You can write your own networkd units to replace or override the units created for each platform. This article covers a subset of networkd functionality. You can view the [full docs here](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html).
+CoreOS machines are preconfigured with [networking customized](notes-for-distributors.md) for each platform. You can write your own networkd units to replace or override the units created for each platform. This article covers a subset of networkd functionality. You can view the [full docs here](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html).
```

Comment:
By default, CoreOS networking is configured for each deployment target or cloud provider. This document explains how to extend or replace those defaults by creating `networkd` units to describe your network.

This document covers only the relevant subset of `networkd` features. See the [full `networkd` documentation](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html) for more.